test/tools/libtesteth/Options.cpp

```diff
@@ -127,12 +127,12 @@ Options::Options(int argc, const char** argv)
         if (arg == "--help")
         {
             printHelp();
-            exit(0);
+			return;
         }
         else if (arg == "--version")
         {
             printVersion();
-            exit(0);
+			return;
```

Comment:
Make sure that the process terminates after the execution takes this path.

Reply:
You can throw an exception instead of silently returning.